plugins/utils/baleia.nix

```diff
@@ -0,0 +1,32 @@
+{
+  helpers,
+  pkgs,
+  config,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config
+{
+  name = "baleia";
+  originalName = "baleia.nvim";
+  defaultPackage = pkgs.vimPlugins.baleia-nvim;
+
+  maintainers = [helpers.maintainers.alisonjenkins];
+
+  settingsOptions = {
+    async = helpers.defaultNullOpts.mkBool true "highlight asynchronously";
+    colors = helpers.defaultNullOpts.mkStr "NR_8" "table mapping 256 color codes to vim colors";
+    line_starts_at = helpers.defaultNullOpts.mkInt 1 "at which column start colorizing";
+    log = helpers.defaultNullOpts.mkEnum ["ERROR" "WARN" "INFO" "DEBUG"] "INFO" "log level, possible values are ERROR, WARN, INFO or DEBUG";
+    name = helpers.defaultNullOpts.mkStr "BaleiaColors" "prefix used to name highlight groups";
+    strip_ansi_codes = helpers.defaultNullOpts.mkBool true "remove ANSI color codes from text";
+  };
```

Comment:
Final nit picking:

Add a blankline between each option definition.
Apart from when the description is very short, prefer this style:

```nix
foo = helpers.defaultNullOpts.mkBool true ''
  Description starts with a capital letter and end with a period.
''
```

This ensures a bit more consistency across the codebase :)

Reply:
Reformatted, please recheck to see if I understood correctly 😄 .

Reply:
Note alejandra broke it up a bit more than I did initially.